.github/workflows/rnmobile-ios-runner.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       run: sudo xcode-select --switch /Applications/Xcode_12.app
```

Comment:
Should we maybe also try to switch to the latest Xcode/simulator if `appium` allows it?

Reply:
Yeah good point, we could try to update it and see if it also works. If you don't mind, I'd like to do it in a separate PR because this PR is blocking [another one](https://github.com/WordPress/gutenberg/pull/29569) that blocks a [second one](https://github.com/WordPress/gutenberg/pull/29309) 😄 .

Reply:
I'm going to test if it works with Xcode version `12.2` (commented [here](https://github.com/WordPress/gutenberg/pull/29589#issuecomment-793642992)).

Reply:
It worked with Xcode version `12.2` ([comment](https://github.com/WordPress/gutenberg/pull/29589#issuecomment-793747731)).

@ceyhun I'm going to merge this PR to unblock the other referenced PRs but we could test with [the latest Xcode version `12.4` supported by MacOS machines](https://github.com/actions/virtual-environments/blob/macOS-10.15/20210302.1/images/macos/macos-10.15-Readme.md#xcode) in a new branch/PR.